packages/apps/stock/src/components/organisms/StockMoveHeader/StockMoveHeader.tsx

```diff
@@ -65,43 +89,43 @@ const StockMoveHeader = ({
           </Text>
         )}
       </View>
-      <View style={styles.badgeContainer}>
+      <View style={styles.badgesContainer}>
         <Badge
           color={getItemColor(StockMove?.statusSelect, status)}
           title={getItemTitle(StockMove?.statusSelect, status)}
         />
-        {availability == null || availability === 0 ? (
-          <View style={styles.refContainer} />
-        ) : (
+        <View style={styles.rowContainer}>
+          {Number(checkQtyObject?.missingQty ?? 0) !== 0 && (
+            <Badge
+              color={Colors.errorColor}
+              title={formatNumber(checkQtyObject?.missingQty)}
+            />
+          )}
           <Badge
             color={getItemColor(StockMove?.availableStatusSelect, availability)}
-            title={getItemTitle(StockMove?.availableStatusSelect, availability)}
+            title={
+              checkQtyObject?.availability ??
+              getItemTitle(StockMove?.availableStatusSelect, availability)
+            }
           />
```

Comment:
This need to be hidden when availableStatusSelect is null or 0 (example on Supplier arrival)